lib/utils/dateOfBirth.ts

```diff
@@ -0,0 +1,21 @@
+import { Applicant } from "../../domain/HousingApi";
+
+/**
+ * Get the applicants age as a number
+ * @param {Applicant} applicant The applicant
+ * @returns {string}
+ */
+export const getAgeInYears = (
+  applicant: Applicant
+): number => {
+  const dateString = applicant.person?.dateOfBirth ?? '';
+  var today = new Date();
+  var dateOfBirth = new Date(dateString);
```

Comment:
Will there be instances of the date of birth being an empty string? Empty strings into dates should return NaN